client/src/components/layout/TopBar.js

```diff
@@ -28,6 +28,9 @@ const TopBar = ({ user }) => {
           <li>
             <Link to="/">Home</Link>
           </li>
+          <li>
+            <Link to="/profile">Your Profile</Link>
```

Comment:
if the profile is an authed route (meaning a user needs to be signed in) then this link should be included with the `authenticatedListItems` to ensure it only appears when a user is signed in